study.md

```diff
@@ -66,7 +66,7 @@ Which ActiveRecord method finds all of the records of a certain type (or
 entity)?
 
 ```md
-<!-- your response here -->
+<!-- .where -->
```

Comment:
`#where` will return everything if no filters are passed into it.  `#all` will return all records of a certain type.